bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java

```diff
@@ -173,8 +173,14 @@ protected HmDatapoint assembleDatapoint(String name, String unit, String type, S
         }
 
         HmValueType valueType = HmValueType.parse(type);
-        if (valueType == null || valueType == HmValueType.UNKNOWN) {
```

Comment:
I would really prefer to always include the braces

Reply:
I did not want to touch existing coding, but I agree with you and will add braces around the throw.

Reply:
done